config/crd/kustomization.yaml

```diff
@@ -6,9 +6,9 @@ resources:
   - bases/core.openfunction.io_servings.yaml
   - bases/core.openfunction.io_builders.yaml
   - bases/events.openfunction.io_eventsources.yaml
-  - bases/events.openfunction.io_eventbus.yaml
+  - bases/events.openfunction.io_eventbuses.yaml
```

Comment:
I cannot find the new `bases/events.openfunction.io_eventbuses.yaml`

Reply:
The file already exists.

https://github.com/OpenFunction/OpenFunction/blob/main/config/crd/bases/events.openfunction.io_clustereventbuses.yaml